frontend/src/app/pages/exercises/exercise/shared/transfer-overview/transfer-time-input/transfer-time-input.component.ts

```diff
@@ -40,4 +40,13 @@ export class TransferTimeInputComponent {
             elementId: this.elementId,
         });
     }
+
+    public transferArriveNow() {
```

Comment:
maybe make this name consistent to `letElementArrive` (maybe even using the same name - could be confusing?)?